build.cmd

```diff
@@ -22,6 +22,29 @@ if not defined VisualStudioVersion (
     exit /b 1
 )
 
+REM Process arguments passed in to build.cmd
+set "__args= %*"
+set __ConfigurationGroup=Debug
+set processedArgs=
+set __Platform=AnyCPU
+:Arg_Loop
+if "%1" == "" goto ArgsDone
+
+if /i "%1" == "-release"                 (set __ConfigurationGroup=Release&set processedArgs=!processedArgs! %1&shift&goto Arg_Loop)
+if /i "%1" == "-debug"                 (set __ConfigurationGroup=Debug&set processedArgs=!processedArgs! %1&shift&goto Arg_Loop)
+if /i "%1" == "-platform"                 (set __Platform=%2&set processedArgs=!processedArgs! %1 %2&shift&shift&goto Arg_Loop)
```

Comment:
What do we pass in platform for?

Reply:
I'm not sure why we pass it, but it's in our build definition. From what I can tell, we always seem to pass `AnyCPU`